.eslintrc.js

```diff
@@ -0,0 +1,387 @@
+  'prefer-spread': 0,
+  'prefer-template': 0,
+  'require-yield': 0,
+  'rest-spread-spacing': 1,
+  'sort-imports': 0,
+  'symbol-description': 0,
+  'template-curly-spacing': 1,
+  'yield-star-spacing': 0,
+
+  // React rules
+  // https://github.com/yannickcr/eslint-plugin-react#list-of-supported-rules
+  'react/boolean-prop-naming': 0,
+  'react/button-has-type': 0,
+  'react/default-props-match-prop-types': 0,
+  'react/destructuring-assignment': 0,
+  'react/display-name': 0,
+  'react/forbid-component-props': 0,
+  'react/forbid-dom-props': 0,
+  'react/forbid-elements': 0,
+  'react/forbid-prop-types': 0,
+  'react/forbid-foreign-prop-types': 0,
+  'react/no-access-state-in-setstate': 0,
+  'react/no-array-index-key': 0,
+  'react/no-children-prop': 0,
+  'react/no-danger': 0,
+  'react/no-danger-with-children': 0,
+  'react/no-deprecated': 0,
+  'react/no-did-mount-set-state': 0,
+  'react/no-did-update-set-state': 0,
+  'react/no-direct-mutation-state': 0,
+  'react/no-find-dom-node': 0,
+  'react/no-is-mounted': 0,
+  'react/no-multi-comp': 0,
+  'react/no-redundant-should-component-update': 0,
+  'react/no-render-return-value': 0,
+  'react/no-set-state': 0,
+  'react/no-typos': 0,
+  'react/no-string-refs': 0,
+  'react/no-this-in-sfc': 0,
+  'react/no-unescaped-entities': 0,
+  'react/no-unknown-property': 0,
+  'react/no-unsafe': 0,
+  'react/no-unused-prop-types': 0,
+  'react/no-unused-state': 0,
+  'react/no-will-update-set-state': 0,
+  'react/prefer-es6-class': 0,
+  'react/prefer-stateless-function': 0,
+  'react/prop-types': 0,
+  'react/react-in-jsx-scope': 0,
+  'react/require-default-props': 0,
+  'react/require-optimization': 0,
+  'react/require-render-return': 0,
+  'react/self-closing-comp': 0,
+  'react/sort-comp': 0,
+  'react/sort-prop-types': 0,
+  'react/style-prop-object': 0,
+  'react/void-dom-elements-no-children': 0,
+
+  // JSX rules
+  // https://github.com/yannickcr/eslint-plugin-react#jsx-specific-rules
+  'react/jsx-boolean-value': 0,
+  'react/jsx-child-element-spacing': 0,
+  'react/jsx-closing-bracket-location': 0,
+  'react/jsx-closing-tag-location': 0,
+  'react/jsx-curly-spacing': 1,
+  'react/jsx-equals-spacing': 1,
+  'react/jsx-filename-extension': [1, { extensions: ['js'] }],
+  'react/jsx-first-prop-new-line': 0,
+  'react/jsx-handler-names': 0,
+  'react/jsx-indent': 0,
+  'react/jsx-indent-props': 0,
+  'react/jsx-key': 0,
+  'react/jsx-max-depth': 0,
+  'react/jsx-max-props-per-line': 0,
+  'react/jsx-no-bind': 0,
+  'react/jsx-no-comment-textnodes': 1,
+  'react/jsx-no-duplicate-props': 0,
+  'react/jsx-no-literals': 0,
+  'react/jsx-no-target-blank': 0,
+  'react/jsx-no-undef': 0,
+  'react/jsx-one-expression-per-line': 0,
+  'react/jsx-curly-brace-presence': 0,
+  'react/jsx-fragments': 0,
+  'react/jsx-pascal-case': 1,
+  'react/jsx-props-no-multi-spaces': 1,
+  'react/jsx-sort-default-props': 0,
+  'react/jsx-sort-props': 0,
+  'react/jsx-space-before-closing': 1,
```

Comment:
This rule is deprecated and replaced by `react/jsx-tag-spacing`, let's remove this line.
https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-space-before-closing.md

Reply:
Removed